src/index.html

```diff
@@ -11,7 +11,147 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    >
```

Comment:
looks easier to read this way, doesn't it?
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="post"
    >
```